src/app/(private)/emergency-evacuation-applications/page.tsx

```diff
@@ -44,53 +46,59 @@ const Page = () => {
       sourceDistrict: search.sourceDistrict,
       targetCity: search.targetCity,
       targetDistrict: search.targetDistrict,
-      isInPerson: search.isInPerson
+      isInPerson: search.isInPerson,
     })
       .then((responseData) => {
         setData(responseData.data.response)
       })
-      .catch((error) => {
-        setError(error.message)
+      .catch(() => {
         toast({
           title: t('error'),
           description: t('defaultError'),
-          variant: 'destructive'
+          variant: 'destructive',
         })
       })
       .finally(() => setIsLoading(false))
+  }
+
+  useEffect(() => {
+    fetchData()
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    JSON.stringify(search)
-  ])
+  }, [searchParamsString])
 
   const { table } = useDataTable({
     data: data.content,
     columns,
     pageCount: data.totalPageCount,
-    filterFields
+    filterFields,
   })
   return (
     <PrivateRoute requiredPermissions={[Permission.EVACUATION_LIST]}>
       <div className="space-y-1">
-        {error && <Toaster />}
```

Comment:
Buradaki Toaster componenti olmadan mesaj görüntülenme yapılamıyor. ayrıca error şartına gerek yok sadece Toaster componentı eklenebilir.

Reply:
App root layout.tsx içerisinde 1 kere ekli olması yeterli. Tekrar alt sayfalarda çağırmaya gerek yok.

Reply:
Yeterli gelmiyor, sayfa içerisine ekleme yapılmadığında toast mesajlar görüntülenmiyor, bu sabah yaşadım :)

Reply:
Tekrar kontrol ettim bir sorun yok , aksine bu şekilde inspector penceresinde aynı elementin kopyası oluşmasına sebep oluyor , shadcn için önerilen kullanım şeklide root layout içerisinde bir kere tanımlanması. Kontrol edip tekrar bakabilirz

Reply:
olur bir araya gelelim müsait olunca

Reply:
cevabınızı bekliyorum ^^

Reply:
bu PR'da tek sorun bu idi, artık sorun değil please approve